src/extensions/http_api/definitions.js

```diff
@@ -125,11 +125,18 @@ exports.install = ({ baseUrl = '' } = {}) => {
         this.httpApiClient.setQuery(Cast.object(this.state.populateObject(step.rowsHash())))
     })
 
-    Given(/^(?:I )?pick response json (.+) as (.+)$/, function (path, key) {
+    /**
+     * Pick a value from previous json response and set it to state
+     */
+    Given(/^(?:I )?pick response json (?:(body|header) )?(.+) as (.+)$/, function (
```

Comment:
Sorry I just noticed something.

We should reword since "json" is a bit weird for header, maybe we could drop "body" for the body case, and alternate between "json" and "header" so that user can write :
* I pick response json foo.bar as fooBar
* I pick response header Location as createdUrl

```suggestion
    Given(/^(?:I )?pick response (json|header) (.+) as (.+)$/, function (
```

Reply:
Thanks for your suggestion. done!